conda_env/cli/main_export.py

```diff
@@ -81,33 +78,14 @@ def configure_parser(sub_parsers):
 
 # TODO Make this aware of channels that were used to install packages
 def execute(args, parser):
-    if not (args.name or args.prefix):
-        # Note, this is a hack fofr get_prefix that assumes argparse results
-        # TODO Refactor common.get_prefix
-        name = os.environ.get('CONDA_DEFAULT_ENV', False)
-        prefix = os.environ.get('CONDA_PREFIX', False)
-        if not (name or prefix):
-            msg = "Unable to determine environment\n\n"
-            msg += textwrap.dedent("""
-                Please re-run this command with one of the following options:
-
-                * Provide an environment name via --name or -n
-                * Re-run this command inside an activated conda environment.""").lstrip()
-            # TODO Add json support
-            raise CondaEnvException(msg)
-        if name:
-            if os.sep in name:
-                # assume "names" with a path separator are actually paths
-                args.prefix = name
-            else:
-                args.name = name
-        else:
-            args.prefix = prefix
-    else:
-        name = args.name
-    prefix = get_prefix(args)
-    env = from_environment(name, prefix, no_builds=args.no_builds,
-                           ignore_channels=args.ignore_channels, from_history=args.from_history)
```

Comment:
@beeankha I realized this is likely redundant given how `determine_target_prefix` is implemented, can you also confirm this before we proceed?

Reply:
Let's go ahead with this code removal! 👍